### PR TITLE
Adjust documentation

### LINKED ISSUE #159: Adjust documenation
# Documentation Update Request

### General remarks

- [x] Please update the Screenshots when the UI is finalized. The current logo is incorrect, and also the wording is not as desired.
- [x] Make sure that the screenshot matches the users persepctive. It is confusing if you are logged in as a servicepartner admin when you have a different view of the portal than a participant while explaining e.g. the User Management from a participant admins point of view. To give an example: In https://github.com/sovity/authority-portal/blob/v2.1.2/docs/product/user-documentation/Administration%20and%20Support.md the Usermanagement is explained but the menu bar shows the item "Provided Connectors" and in your profile the servicepartner role is displayed. Please log in as a particiapant admin with no application role when you are showing features that apply to that role. A different example where this mismatch also occurs is https://github.com/sovity/authority-portal/blob/v2.1.2/docs/product/user-documentation/Connect%20to%20the%20MDS.md
- [x] I'm missing a discription of the toggle bar used for switching the environment. Please add explainations for that (e.g. how to change the environment, which areas of the portal are affected by changing the toggle bar).
- [x] Please use appropriate names for organizations like "Your Organziation" etc. instead of "Evil corp" and make sure your not exposing your company or companies employees. Multiple of your screenshots show your name or the names of other sovity employees. Please try to be neutral here.
- [x] As I already mentioned in https://mobility-dataspace.online/?wpsc-section=ticket-list&ticket-id=188 I would very much prefer it if there was a clear separation between documentation that is relevant for the user and documentation that is relevant for application roles. It is not relevant for a non authority user to know how organizations are invited or rejected. Please separate this.
- [x] I'm missing an overall description of the portal, that also describes how the portal is structured. Maybe such a description could be added to the end of "Registration in the MDS Portal".
- [x] There is no documentation about the dashboard.

### Administration and Support

- [x] The Chapter "User Management" is not intuitivly structured for me. I would start with explaining where a user can see his role, what roles exist and roughly what they can do and then I would go into detail explaining how a participant admin can invite new users and change a users role and status.
- [x] As already mentioned in General remarks the organization management is only relevant for authority-Roles and should therefor not be intertwined with the documentation for the customer. It should be part of a separate file.
- [x] The same is true for the chapter "Connector Overview". This information is only relevant for authority admins, authority users and the operator.

### Connect to the MDS

- [x] You are giving a description on the on premise installation but how to request a CaaS and a short discription about the CaaS is missing. In the chapter "Connector-as-a-Service" in https://github.com/sovity/authority-portal/blob/v2.1.2/docs/product/user-documentation/Connect%20to%20the%20MDS.md you are advicing the user to go to this page https://github.com/Mobility-Data-Space/mobility-data-space/wiki/ (that btw also is about the on prem installation)  while they actually could just order a CaaS through the portal. Could you please explain the CaaS-Ordering process in the portal?
- [x] Please add a screenshot that shows the "add connector" button. You are describing this button but it does not appear in any of the screenshots on that page.

### Registration in MDS Portal

- [x] Wouldn't it make more sense to start with the organization registration and then go over the user invitation as this is the prerequisit for registering a user? From a users perspective there are two ways to get access to the portal: a) by registering his organization if the organization does not exists yet or b) by receiving an invitation from the authority.
- [x] If the organization is already registered the user has to ask his admin to invite him -> reference to the Administration chaptwr where the invite-Process should be explained.

### PR DESCRIPTION
**Pull request by jridderbusch**
closes #159 

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [x] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: docs/product/user-documentation/Connect to the MDS.md [hunk 1]
```diff
@@ -1,49 +1,85 @@
...
 
-![choose-connector](images/choose-connector-registration.png)
+![choose-connector](images/connect-to-the-mds/choose-connector-registration.png)
 
 Fill out the "Add Connector Details"-mask and click "Next" to proceed.
 
-![example-connector-registration](images/example-values-connector-registration.png)
+![register-connector-form](images/connect-to-the-mds/register-connector-form.png)
 
 After entering your Connector details, you now need to give the stated information in the "Generate Certificate"-mask. and click "Generate & Download Certificate".
```
Comment (kamilczaja) [inline docs/product/user-documentation/Connect to the MDS.md:67]: There is a period that shouldn't be there

File: docs/product/user-documentation/images/registration-in-mds-portal/self-registration.png
Comment (kamilczaja) [inline docs/product/user-documentation/images/registration-in-mds-portal/self-registration.png:1]: Screenshot is outdated

**Review by kamilczaja — CHANGES_REQUESTED** [review]

File: docs/product/user-documentation/Authority Section.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- _Active_: Fully registered organizations
+- _Invited_: Organization invitation has been sent, but was not yet accepted
+- _Pending_: Organization has send a registration request
+- _Rejected_: Organization registration request has been rejected
+
+Click in the row of an organization to view the organization details. Quickly switch between two organization detail pages by using the arrows at the top.
+
+![organization-detail-page](images/authority-section/organization-detail-page.png)
+
+From the organization detail page, authority users and admins are able to manage the organization of interest and e.g approve or reject an organization or see the different users of it.
```
[the remark this answers is not in the document]
Comment (jridderbusch) [reply docs/product/user-documentation/Authority Section.md:20]: I was just working through a list of requested changes, I did not read all the documentation and do not intend to do so.

**Review by kamilczaja — APPROVED** [review]
LGTM

**Review by kamilczaja — APPROVED** [review]

**Pull request merged**